### ISSUE TITLE
No red or green status markers in data docs in Great Expectations

### ISSUE THREAD

**Issue by unkind58** [great-expectations/great_expectations#10475]
**Describe the bug**
After running any of my created checkpoints - I could not see `red` or `green` status markers in data docs in Great Expectations:
![image](https://github.com/user-attachments/assets/4d8c3ae0-d9cb-44b8-b8e6-1f05ddf76db2)

The same appears for my collegues and the same appears for older (run, for example 2-5 months ago) data-docs results.

**To Reproduce**
Run any Great Expectations checkpoint and build data-docs. 
```
checkpoint = context.add_or_update_checkpoint(
    name=f"{layer}_{datasource}_checkpoint",
    run_name_template=f"%Y%m%d-%H%M%S-{layer}-{datasource}",
    validations=[
        {
            "batch_request": b_request, # request variable title
            "expectation_suite_name": e_suite, # suite variable title
        },
    ],
    action_list=[
        {
            "name": "store_validation_result",
            "action": {"class_name": "StoreValidationResultAction"},
        },
        {"name": "update_data_docs", "action": {"class_name": "UpdateDataDocsAction"}},
    ],
)
context.add_or_update_checkpoint(checkpoint=checkpoint)
checkpoint_result = checkpoint.run()
context.build_data_docs()
context.open_data_docs()
```

**Expected behavior**
In data-docs rendered _html_, in the STATUS cells you can see `green` (if success: True) and `red` (if success: False).

**Environment (please complete the following information):**
 - Operating System: Windows
 - Great Expectations Version: tried on both 0.18.19 & 0.18.8
 - Data Source: Spark data frames from delta tables and csv files
 - Cloud environment: delta tables and csv files are being loaded from AWS S3

**Comment by leodrivera** [comment]
Have you configured the CSP to allow these contents?

**Comment by nl-hugo** [comment]
Same for me. Status markers were showing just fine until a couple of days ago. Our configuration and GX version are unchanged. 

Upon inspection of the rendered HTML page in the browser, I noticed the following network error:

```
Request URL: https://kit.fontawesome.com/8217dffd95.js
Request Method: GET
Status Code: 403 Forbidden
Remote Address: ...omitted...
Referrer Policy: strict-origin-when-cross-origin
```

**Comment by leodrivera** [comment]
I've checked here and the js link is indeed broken

**Comment by BONDSK01** [comment]
I have same issue. 

I'm gettting 
`Request URL:
https://kit.fontawesome.com/8217dffd95.js
Request Method:
GET
Status Code:
403 Forbidden`

all of a sudden.

**Comment by BONDSK01** [comment]
Also, to reproduce all I have to do is open any index.html I've previously created (that were showing the status icon previous to 10/4) and I no longer see icons on any of those either.

**Comment by dkt-sophie-ly** [comment]
Same issue here :/

I need to add this in all the html file generated to make it works:

`<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">`

**Comment by BONDSK01** [comment]
@dkt-sophie-ly - How did you find this partial solution? This will get overwritten after each test thought correct?

**Comment by Erua-chijioke** [comment]
I have the same issue and I am using GX version 1.1.0

**Comment by dkt-sophie-ly** [comment]
> @dkt-sophie-ly - How did you find this partial solution? This will get overwritten after each test thought correct?

Yes correct... 

Opening the html file I notice that this icon is not displaying: fas fa-check-circle text-success 
Then I try this solution: https://stackoverflow.com/questions/53953884/font-awesome-circles-not-showing-up
and it worked for me.

**Comment by BONDSK01** [comment]
Let's be clear. This isn't a solution. The bug still exists and will need to be addressed by the GX team.

**Comment by BONDSK01** [comment]
@tyler-hoffman - I guess everyone will need to upgrade versions to fix the status icon missing bug? Since a new font pack was created? I'm definitely not seeing the status icons in my old docs. Do you know which release this fix will be included in?

**Comment by adeola-ak** [comment]
@BONDSK01 hi there, this fix for this issue was released yesterday with version 1.1.3

**Comment by kujaska** [comment]
> @BONDSK01 hi there, this fix for this issue was released yesterday with version 1.1.3

@adeola-ak  any ideas how it could be quickly fixed WITHOUT upgrade to GE v.1.1.3?

There are a LOT of installations of older versions in the world and upgrade to v.1+ is not easy due to API changes... 
Furthermore, some environments like old on-prem even CAN'T upgrade to v.1+ due to dependency limitations (e.g. still running on Python 3.9)

**Comment by adeola-ak** [comment]
Unfortunately, versions below v1+ are no longer supported. I highly recommend upgrading, as older versions will not receive support for any issues. We have a [migration guide](https://docs.greatexpectations.io/docs/reference/learn/migration_guide/?utm_campaign=General%20Community%20Promotion&utm_source=GitHub&utm_medium=post&utm_content=migration-guide) that is quite intuitive and should help make the upgrade process smoother.

If upgrading isn’t an option at the moment, one workaround could be to use a local Font Awesome kit and place it in a local folder within your project, and then update the Data Docs HTML template to reference the local version instead of the external link that’s causing the 403 error. Just a thought

**Comment by kujaska** [comment]
@adeola-ak update the Data Docs HTML template to reference the local version 

how to do it? where is the reference?

**Comment by adeola-ak** [comment]
looking at the fix in the linked pull request, it looks like its just a matter of changing the script tag in `great_expectations/render/view/templates/js_script_imports.j2` to use the new kit.

**Comment by kujaska** [comment]
Great news - fixed in 0.18.22 !!!
Many Thanks!
